From 0e415985daa1c0e8c029a6a9a58ccfbd6341022b Mon Sep 17 00:00:00 2001
From: Stefan Becker <chemobejk@gmail.com>
Date: Thu, 3 Aug 2017 12:05:45 +0300
Subject: [PATCH] package/skeleton-common: fix recursive variable

This fixes a recursive variable definition introduced by the refactoring
in commit 120307520f69f801bf05df1d2fd3433c6d498551. Simply rename the
hook to remove the recursion.

Fixes https://bugs.busybox.net/show_bug.cgi?id=10146

Signed-off-by: Stefan Becker <chemobejk@gmail.com>
---
 package/skeleton-common/skeleton-common.mk | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/package/skeleton-common/skeleton-common.mk b/package/skeleton-common/skeleton-common.mk
index e9b87c3..e94dd6e 100644
--- a/package/skeleton-common/skeleton-common.mk
+++ b/package/skeleton-common/skeleton-common.mk
@@ -75,16 +75,16 @@ endef
 SKELETON_COMMON_TARGET_FINALIZE_HOOKS += SKELETON_COMMON_SET_ROOT_PASSWD
 
 ifeq ($(BR2_SYSTEM_BIN_SH_NONE),y)
-define SKELETON_COMMON_BIN_SH
+define SKELETON_COMMON_SET_BIN_SH
 	rm -f $(TARGET_DIR)/bin/sh
 endef
 else
 ifneq ($(SKELETON_COMMON_BIN_SH),)
-define SKELETON_COMMON_BIN_SH
+define SKELETON_COMMON_SET_BIN_SH
 	ln -sf $(SKELETON_COMMON_BIN_SH) $(TARGET_DIR)/bin/sh
 endef
 endif
 endif
-SKELETON_COMMON_TARGET_FINALIZE_HOOKS += SKELETON_COMMON_BIN_SH
+SKELETON_COMMON_TARGET_FINALIZE_HOOKS += SKELETON_COMMON_SET_BIN_SH
 
 $(eval $(generic-package))
-- 
2.10.1

